tests.yaml

```diff
@@ -0,0 +1,239 @@
+                home: https://github.com/CKI-project/tests-beaker/tree/master/packages/iotop/sanity
+    fs:
+        title: Filesystem tests
+        tests:
+            xfstests:
+                title: Xfstests filesystem testing suite
+                home: https://github.com/kdave/xfstests
+            pjdfstest:
+                title: A test suite for exercising filesystem-oriented POSIX system calls
+                home: https://github.com/pjd/pjdfstest
+            redhat_cifs_connectathon:
+                title: Red Hat version of LTP's NFS Connectathon modified to test CIFS
+                home: https://github.com/CKI-project/tests-beaker/tree/master/filesystems/cifs/connectathon
+            redhat_loopdev:
+                title: Red Hat's loop device sanity tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/filesystems/loopdev/sanity
+    net:
+        title: Network-related tests
+        tests:
+            redhat_ethernet:
+                title: Red Hat's Ethernet driver sanity tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/driver/sanity
+            redhat_l2tp:
+                title: Red Hat's basic L2TP tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/tunnel/l2tp/basic
+            redhat_macsec:
+                title: Red Hat's sanity test for MACsec support
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/macsec/sanity_check
+            redhat_tcp_keepalive:
+                title: Red Hat's tests for TCP Keep-Alive support
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/tcp/tcp_keepalive
+            redhat_udp_socket:
+                title: Red Hat's tests for UDP socket support
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/udp/udp_socket
+            redhat_bridge:
+                title: Red Hat's Ethernet bridge tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/bridge/sanity_check
+            redhat_ipsec:
+                title: Red Hat's IPSec tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/ipsec/ipsec_basic/ipsec_basic_netns
+            redhat_pmtu:
+                title: Red Hat's tests for Path MTU discovery
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/route/pmtu
+            redhat_route:
+                title: Red Hat's IP routing tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/route/route_func
+            redhat_sctp_auth_sockopts:
+                title: Red Hat's SCTP authentication socket options tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/sctp/auth/sockopts
+            redhat_sock_fuzz:
```

Comment:
`socket_fuzz` would seem more consistent with how the others are named, reusing the path `socket/fuzz`.  I don't mind really, but if it caught my eye it might catch others' too.

Reply:
Yeah, it could be clearer to name it "redhat_socket_fuzz", indeed. I'll change that.